src/solver/include/grins/mesh_adaptive_solver_base.h

```diff
@@ -72,6 +72,7 @@ namespace GRINS
     libMesh::Real _refine_fraction;
     libMesh::Real _coarsen_fraction;
     libMesh::Real _coarsen_threshold;
+    bool compute_QoI_error_estimate;
```

Comment:
Please use consistent naming. For members of a class, the name is prefixed with a `_` and all the characters are lower case.